core/evalir.ml

```diff
@@ -601,7 +620,11 @@ struct
           let sort = Type.sort t in
           value env table >>= fun table ->
           match table with
-            | `Table (((db,cstr), table, _, _) as tinfo) ->
+            | `Table tinfo ->
+              (* I can't get this to work without turning the warning off, even
+                 with "Value" in scope. *)
+              let [@warning "-40"] (db, cstr) = tinfo.database in
```

Comment:
Disabling this warning is unacceptable! The correctness of this code now depends on the order of type inference. It is really bad and it **will** come back to bite us later. It can be difficult to understand the origin of these warnings, however, this particular case is because of heavy re-use of module names and record labels. The correct thing to do here is to make type disambiguation possible. To make it possible you must introduce namespaces.

I will fix these instances for you. Next time please ask for help.

Reply:
This was a temporary workaround for issues encountered while merging, which I didn't intend to be in the PR. Apologies. Thank you for sorting it.